src/utils/ValidationUtil.ts

```diff
@@ -0,0 +1,2 @@
+export const checkBlankSpace = (value: string) => !!value.match(/\S/g);
+export const checkIsUrl = (value: string) => !!value.match(/^https?:\/\/.+$/g);
```

Comment:
```suggestion
export const checkIsUrl = (value: string) => /^https?:\/\/.+$/.test(value);
```
`test` を使いましょう
あと、`^` と `$` を使っているのにグローバル指定にしているのが正しいのか分からない